group-coordinator/src/test/java/org/apache/kafka/coordinator/group/OffsetMetadataManagerTest.java

```diff
@@ -2576,15 +2577,15 @@ public void testCleanupExpiredOffsetsWithPendingTransactionalOffsets() {
 
         OffsetMetadataManagerTestContext context = new OffsetMetadataManagerTestContext.Builder()
             .withGroupMetadataManager(groupMetadataManager)
-            .withOffsetsRetentionMs(1000)
+            .withOffsetsRetentionMinutes(1)
```

Comment:
not sure why we need this change. Also, the value is changed from 1 second to 1 minute?

Reply:
Actually the original time unit of config `offsets.retention.minutes` is in minutes, and before this refactor the we pass `offsets.retention` in millis to GroupCoordinatorConfig constructor and use it in OffsetMetadataManagerTest, now we pass AbstractConfig (i.e. KafkaConfig) to GroupCoordinatorConfig, which means we should pass offsets.retention in minutes instead of ms